x/auth/vesting/msg_server_test.go

```diff
@@ -0,0 +1,257 @@
+package vesting_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
```

Comment:
```suggestion
	"github.com/golang/mock/gomock"
	"github.com/stretchr/testify/suite"
	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
	tmtime "github.com/tendermint/tendermint/types/time"
```

Reply:
It will be rearranged by vscode Go extension. It can be messy to check imported modules.